runtime/cmake/Modules/Findnccl.cmake

```diff
@@ -0,0 +1,66 @@
+## Copyright (c) Facebook, Inc.
+## Licensed under BSD License
+## ===========================================================================
+## Modification Copyright 2022 ByteDance Ltd. and/or its affiliates.
```

Comment:
@yaochengji  We need to consider changing the " Copyright 2022" to "Copyright 2023"